Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/IntakeEject.java

```diff
@@ -21,6 +22,9 @@
 	private final DoubleSolenoid leftHorizontalSolenoid = RobotMap.leftIntakeHorizontalSolenoid;
 	private final DoubleSolenoid rightHorizontalSolenoid = RobotMap.rightIntakeHorizontalSolenoid;
 
+	private boolean leftOpen = false;
+	private boolean rightOpen = leftOpen;
+
```

Comment:
Why do we need these? Can't we compute the state using the values of leftHorizontalSolenoid.get() and rightHorizontalSolenoid.get()?

Reply:
Done in latest commit.